src/components/AppForm.js

```diff
@@ -0,0 +1,92 @@
+import { LitElement, html, css } from 'lit-element';
+
+import './AppButton';
+
+class AppForm extends LitElement {
+
+  static get styles() {
+    return css`
+      .wrapper {
+        padding: 20px;
+        border: 1px solid #eee;
+        background-color: #fff;
+        border-radius: 6px;
+      }
+      div {
+        padding: 6px 0;
+      }
+      input[type="text"], textarea {
+        width: 100%;
+        padding: 6px 8px;
+        box-sizing: border-box;
+      }
+      textarea {
+        resize:vertical;
+      }
+    `;
+  }
+
+  constructor() {
+    super();
+
+    this.title = '';
+    this.description = '';
```

Comment:
I see these props being initialized here but there is no props declaration. Why is that so?